src/pages/Login.tsx

```diff
@@ -0,0 +1,23 @@
+import AppBackButton from "../components/atom/AppBackButton";
+import LoginForm from "../components/molecules/LoginForm";
+
+const Login = () => {
+    return (
+        <div className="h-screen flex flex-col page-bg text-blue-dark p-8 justify-between">
+            <div className="flex flex-col gap-8 self-center md:w-1/2 lg:w-1/3">
+                <AppBackButton goTo="/password-recovery"/>
+                <div>
+                    <h1 className="text-1xl font-bold mb-4">
+                        Inicia sesión
```

Comment:
Would it be possible to have some i18n file already, no matter if mocked or with just 1 idiom, so we can identify later all the texts to be translated/defined?